app/src/main/java/com/simplecity/amp_library/ui/settings/SupportPresenter.java

```diff
@@ -53,6 +53,16 @@ public void helpClicked() {
         }
     }
 
+    public void emailClicked() {
+        Intent intent = new Intent(Intent.ACTION_SENDTO);
+        intent.setData(Uri.parse("mailto:"));
+        intent.putExtra(Intent.EXTRA_EMAIL, "pal-muser@usf.edu");
```

Comment:
This email isn't being auto-populated for some reason when I tap on "Email us". Gmail opens directly, but no email is in the "To:" field. Samsung Galaxy S8+ w/ Android 9.

Here's code we've used in OneBusAway:

```java
String email = conext.getString(R.string.email);
Intent send = new Intent(Intent.ACTION_SEND);
        send.putExtra(Intent.EXTRA_EMAIL,
                new String[]{email});
        String subject = context.getString(R.string.bug_report_subject);
        String body = "xxxxxxxxxxxxxxx";
        send.putExtra(Intent.EXTRA_SUBJECT, subject);
        send.putExtra(Intent.EXTRA_TEXT, body);
        send.setType("message/rfc822");
        try {
            context.startActivity(Intent.createChooser(send, subject));
        } catch (ActivityNotFoundException e) {
            Toast.makeText(context, R.string.bug_report_error, Toast.LENGTH_LONG)
                    .show();
        }
```

The email address should be moved to a `donottranslate.xml` file under the `values` resources folder.

Also, we should populate the subject line with something like "MUSER Feedback"